CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Add hide history option when adding a new member [#2155](https://github.com/GetStream/stream-chat-swift/issues/2155)
 - Add Extra Data Usage Improvements [#2174](https://github.com/GetStream/stream-chat-swift/pull/2174)
   - For more details please read the documentation [here](https://getstream.io/chat/docs/sdk/ios/uikit/extra-data).
+- Support for message moderation (NNBB) [#2103](https://github.com/GetStream/stream-chat-swift/pull/2103/files)
```

Comment:
We need to wait for release 4.19.0 to come to develop, so that we can add this one to the #Upcoming release. So please don't merge until we have the 4.19.0 on develop 👍